README.md

```diff
@@ -67,17 +67,17 @@ Review our [Contributing Instructions](CONTRIBUTING.md) before beginning editing
-| 52 | 2022-xx-xx | William Lyon| [Introduction to GraphQL for Data Scientists](2022/52-will-graphql.md)|
-| 53 | 2022-xx-xx | Sebastian Raschka and Adrian Wälchli| [Introduction to PyTorch](2022/53-sebastian-adrian-pytorch.md) | 
-| 54 | 2022-xx-xx | Melissa Weber Mendonça | [Contributing to SciPy](2022/54-melissa-scipy.md) | 
-| 55 | 2022-xx-xx | Joe Torreggiani | [Intro to Django](2022/55-joe-django.md) | 
-| 56 | 2022-xx-xx | Tereza Iofciu | [Short Stories of Data Visualization](2022/56-tereza-visualization.md) | 
+| 51 | 2022-04-05 | Rob Lanphier| [Editing Wikipedia: Because Someone Has to...](2022/51-rob-wikipedia.md)| [v51_Editing_Wikipedia](https://www.youtube.com/watch?v=s5kUSMUaZLM)                     |              |        |
+| 52 | 2022-04-26 | William Lyon| [Introduction to GraphQL for Data Scientists](2022/52-will-graphql.md)| [v52_GraphQL_for_Data_Scientists](https://youtu.be/3QVguMgyAhU)
+| 53 | 2022-05-10 | Sebastian Raschka and Adrian Wälchli| [Introduction to PyTorch](2022/53-sebastian-adrian-pytorch.md) | [v53_PyTorch_introduction](https://www.youtube.com/watch?v=B5GHmm3KN2A)
+| 54 | 2022-05-24 | Melissa Weber Mendonça | [Contributing to SciPy](2022/54-melissa-scipy.md) | [v54_SciPy_contributions](https://youtu.be/V1FCSijy460)
+| 55 | 2022-06-07 | Joe Torreggiani | [Intro to Django](2022/55-joe-django.md) | [v55_Django_intro](https://youtu.be/Yr1ewxg8os8)
+| 56 | 2022-06-14 | Tereza Iofciu | [Short Stories of Data Visualization](2022/56-tereza-visualization.md) | [v56_data_visualization](https://youtu.be/bqZpYY0XBT8)
 | 57 | 2022-xx-xx | Data Umbrella & PyMC | [Data Umbrella & PyMC Sprint](2022/57-du-pymc-sprint.md) |
-| 58 | 2022-xx-xx | Emily Lescak & Diego Saez-Trumper | [How Data Scientists Can Contribute To Wikimedia Projects](2022/58-emily-diego-wikimedia.md) |
-| 59 | 2022-xx-xx | Nidhin Pattaniyil & Vishal Rathi | [Serving PyTorch Models in Production](2022/59-nidhin-pytorch.md) |
-| 60 | 2022-xx-xx | Abdulazeez Abdulazeez Adeshina | [Building APIs with FastAPI](2022/60-abdulazeez-fastapi.md) |
-| 61 | 2022-xx-xx | Joe Torreggiani | [Intro to Front-end Development with React](2022/61-joe-react.md) |
+| 58 | 2022-07-12 | Emily Lescak & Diego Saez-Trumper | [How Data Scientists Can Contribute To Wikimedia Projects](2022/58-emily-diego-wikimedia.md) | [v58_Wikimedia_project_contributions](https://youtu.be/mrHp3wc_6DQ)
+| 59 | 2022-07-16 | Nidhin Pattaniyil & Vishal Rathi | [Serving PyTorch Models in Production](2022/59-nidhin-pytorch.md) | [v59_PyTorch_in_production](https://youtu.be/fx_NaKwFYbg)
+| 60 | 2022-08-10 | Abdulazeez Abdulazeez Adeshina | [Building APIs with FastAPI](2022/60-abdulazeez-fastapi.md) | [v60_API_with_FastAPI](https://youtu.be/aGbUwDYBMhs)
+| 61 | 2022-08-16 | Joe Torreggiani | [Intro to Front-end Development with React](2022/61-joe-react.md) | [v61_frontend_intro](https://youtu.be/jNO-pPR7zkg)
```

Comment:
This event, in fact, has 3 videos.
We should include all 3 videos here, OR one playlist.